docs/examples/Polar_Plotting.py

```diff
@@ -0,0 +1,197 @@
+from stonesoup.types.array import StateVector
+from stonesoup.types.groundtruth import GroundTruthPath
+from stonesoup.types.state import State
+
+# Define the simulation start time
+start_time = datetime(2023, 1, 1)
+
+# %%
+# Generate Cartesian State Space Data
+# -----------------------------------
+# Two targets are created:
+#  #. Target 1 moves in a ‘C’ shape. First it moves west (negative x), then it starts a slow,
+#     long 180-degree turn moving south, until it is moving east (positive x).
+#  #. Target 2 moves in a straight line from north to south.
+
+# Create manoeuvre behaviours and durations for our moving platform
+straight_level = CombinedLinearGaussianTransitionModel(
+    [ConstantVelocity(0.), ConstantVelocity(0.)])
+
+# Configure target 1's turn behaviour
+turn_noise_diff_coeffs = np.array([0., 0.])
+turn_rate = np.deg2rad(5)  # specified in radians per second
+turn_model = KnownTurnRate(turn_noise_diff_coeffs=turn_noise_diff_coeffs, turn_rate=turn_rate)
+turning = CombinedLinearGaussianTransitionModel([turn_model])
+
+manoeuvre_list = [straight_level, turning, straight_level]
+manoeuvre_times = [timedelta(seconds=12),
+                   timedelta(seconds=36),
+                   timedelta(seconds=12)]
+
+# %%
+# Using the manoeuvres created previously, two platforms are created that operate in Cartesian
+# state space.
+target_1_initial_state = State(StateVector([[600], [-50], [600], [0]]), start_time)
+target_1 = MultiTransitionMovingPlatform(transition_models=manoeuvre_list,
+                                         transition_times=manoeuvre_times,
+                                         states=target_1_initial_state,
+                                         position_mapping=(0, 2),
+                                         velocity_mapping=(1, 3),
+                                         sensors=None)
+
+target_2_initial_state = State(StateVector([[500], [0], [700], [-24]]), start_time)
+target_2 = MovingPlatform(transition_model=straight_level,
+                          states=target_2_initial_state,
+                          position_mapping=(0, 2),
+                          velocity_mapping=(1, 3),
+                          sensors=None)
+
+# %%
+# Simulate platform movement:
+timesteps = [start_time + timedelta(seconds=i) for i in range(60)]
+
+for t in timesteps:
+    target_1.move(t)
+    target_2.move(t)
+
+# %%
+# Display ground truth in Cartesian state space using the standard
+# :class:`~.Plotterly` plotting class:
+plotter_xy = Plotterly(title="Bird's Eye View of Targets")
+mapping = [0, 2]
+plotter_xy.plot_ground_truths(target_1, mapping=[0, 2], truths_label="Target 1")
+plotter_xy.plot_ground_truths(target_2, mapping=[0, 2], truths_label="Target 2")
+plotter_xy.fig
+
+# %%
+# Generate Angular State Space Data
+# ----------------------------------
+# Measure the previously generated cartesian state space using a
+# :class:`~.RadarElevationBearingRange` sensor:
```

Comment:
The sensor used in this example is the RadarBearingRange sensor, rather than RadarElevationBearingRange.
```suggestion
# :class:`~.RadarBearingRange` sensor:
```